lib/Service/OpenProjectAPIService.php

```diff
@@ -454,6 +455,56 @@ public static function validateURL(string $url): bool {
 			preg_match('/^https?/', $url);
 	}
 
+	/**
+	 * validates the provided data for integration setup
+	 *
+	 * @param array<mixed> $values
+	 * @return bool
+	 * @throws \InvalidArgumentException
+	 *
+	 */
+	public static function validateIntegrationSetupInformation(array $values, ?string $keyType = null): bool {
```

Comment:
actually the second parameter is a bool, it only can have two values
```suggestion
	public static function validateIntegrationSetupInformation(array $values, bool $allKeysMandatory = true): bool {
```